code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -198,8 +198,9 @@
 	..()
 	new /obj/item/weapon/grenade/empgrenade(src)
 	new /obj/item/weapon/grenade/empgrenade(src)
+	new /obj/item/weapon/grenade/empgrenade(src)
+	new /obj/item/weapon/grenade/empgrenade(src)
 	new /obj/item/weapon/implanter/emp(src)
-	new /obj/item/device/flashlight/emp(src)

```

Comment:
i think this is here because i haven't updated my repo in a while, but it's probably safe to ignore